remote-robot/src/main/kotlin/com/intellij/remoterobot/fixtures/Fixture.kt

```diff
@@ -57,6 +62,17 @@ abstract class Fixture(
         return data.getMany { textPredicate.test(it) }
     }
 
+    /**
+     * Get component screenshot.
+     * Use ImageIO.write() method with "png" formatName
+     *
+     * @param withPainting false by default. Set true for rendering the component before capturing screenshot
+     * @return BufferedImage specified as .png
+     */
+    fun getScreenshot(withPainting: Boolean = false): BufferedImage {
```

Comment:
Could you please check that this method is usable from java code for both options